CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.11.1
+
+* Update Conjur Dockerfile from 14.04 --> 18.04.
+  This also required a bump of ruby from 2.2 to 2.3 as 2.2 isn't available in 14.04
```

Comment:
typo 14.04 should be 18.04

Reply:
I think it is ruby-dev that's missing and not ruby itslef https://www.brightbox.com/blog/2015/01/05/ruby-2-2-0-packages-for-ubuntu/

Reply:
Hah, so I was wrong, its not ubuntu, its a brightbox PPA, and their [documentation](https://www.brightbox.com/docs/ruby/ubuntu/) doesn't match reality. 

```
Ruby 2.2

We maintain Ruby 2.2 packages for Ubuntu Bionic, Xenial and Trusty. They include security fixes backported by us.
```

But [launchpad](https://launchpad.net/~brightbox/+archive/ubuntu/ruby-ng?field.series_filter=bionic) shows:
![Screenshot 2019-11-28 at 16 48 31](https://user-images.githubusercontent.com/253506/69822735-036c8e00-11ff-11ea-86a0-644575f357b0.png)


I'll update the messages to clarify the source of packages.